script/Config/App.lua

```diff
@@ -30,17 +30,20 @@ Config.audio = {
     mainMenuMusicEnabled = true,
     soundMin             = 0,
     soundMax             = 1,   -- SetVolume range seems to go from 0 (min) to about 2 or 3 (max)
-    musicVolume          = 0.75, -- current volume
+    musicVolume          = 0.0, -- current volume
```

Comment:
oi :D - we need a menu option for this

Reply:
Agreed. I've added a to-do. In the meantime, I've set this back to 0.75.

I had to delete my user.ini file to get it to "take," but music is playing again.